src/modules/chat/assistant-plan/PlanStep.tsx

```diff
@@ -264,9 +262,9 @@ export function PlanStep({ step, toolCall }: Props) {
                   </div>
                 </div>
               ) : (
-                <AnimatePresence>
```

Comment:
Does it work without the `AnimatePresence`?

Reply:
I was just testing something to get rid of 'unique "key" prop' errors in MotionComponent (I didn't resolve it). As a matter of fact it should only be required to enable exit animations according to the documentation, but it seems that it's not working without it, reverting.